how-to/how-to-add-telerik-ui-for-nativescript-package.md

```diff
@@ -0,0 +1,69 @@
+---
+title: How to Include the {{site.tuin}} Package
+description: A tutorial for adding {{site.tuin}} to a mobile application in AppBuilder. You can use the UI controls by including the free limited edition or by downloading the pro version package from your account and importing it in your app
+type: how-to
+page_title: How to Add {{site.tuin}} in AppBuilder
+slug: include-nativescript-ui-package
+position:
+tags: appbuilder, nativescript
+teampulseid:
+ticketid:
+pitsid:
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code (AppBuilder)</td>
```

Comment:
For later: we should consider mentioning the services and frameworks here using variables as well.